sentences/de-CH/light_HassTurnOff.yaml

```diff
@@ -2,7 +2,12 @@ language: de-CH
 intents:
   HassTurnOff:
     data:
-      - sentences: []
+      - sentences:
+          - "(Zünd|Mach|Lösch) (<Liecht>|<Liechter>) <ab_us> <area>"
+          - "(Zünd|Mach|Lösch) (<Liecht>|<Liechter>) <area> <ab_us>"
+          - "(Zünd|Mach|Lösch ) <area> (<Liecht>|<Liechter>) <ab_us>"
+          - "(Schalt|Schaut) (<Liecht>|<Liechter>) <ab_us> <area>"
+          - "(Schalt|Schaut) (<Liecht>|<Liechter>) <area> <ab_us>"
         slots:
-          domain: light
-          name: all
+          domain: "light"
+          name: "lights_area"
```

Comment:
you should have `response` statement. Which response in `responses` should be used.

Reply:
slots:
    domain: light
  response: lights_area